api/src/main/resources/db/migration/V1.0__create_sсhema.sql

```diff
@@ -18,9 +18,35 @@ create table tag
     constraint unique_tag_name unique (name)
 );
 
-create table certificate_tag
+create table certificate_x_tag
 (
     gift_certificate_id integer not null references gift_certificate on delete cascade,
     tag_id              integer not null references tag on delete cascade,
     primary key (gift_certificate_id, tag_id)
+);
+
+create table consumer
```

Comment:
table consumer - I do not see Java entity for this table.   class Order = table order .  class  OrderItem + table 

I think you should use USER  for this purpose.

Reply:
table name 'user' and 'order' are not allowed cause they are keywords

Reply:
I would use plural:  "users, orders". If so, I'll have to rename all previous tables to plural